packages/actions/components/Stake.tsx

```diff
@@ -46,24 +46,48 @@ export const stakeActions: { type: StakeType; name: string }[] = [
 
 interface StakeOptions {
   nativeBalances: readonly Coin[]
+  nativeDelegatedBalance: Coin
 }
 
 export const StakeComponent: ActionComponent<StakeOptions> = ({
   fieldNamePrefix,
   onRemove,
   errors,
   isCreating,
-  options: { nativeBalances },
+  options: { nativeBalances, nativeDelegatedBalance },
 }) => {
   const { t } = useTranslation()
-  const { register, watch, setError, clearErrors, setValue } = useFormContext()
+  const { register, watch, getValues, setError, clearErrors, setValue } =
+    useFormContext()
 
   const stakeType = watch(fieldNamePrefix + 'stakeType')
   const amount = watch(fieldNamePrefix + 'amount')
   const denom = watch(fieldNamePrefix + 'denom')
 
   const validatePossibleSpend = useCallback(
     (denom: string, amount: string): string | boolean => {
+      const delegateType = getValues(fieldNamePrefix + 'stakeType')
```

Comment:
This value is already retrieved on line 63 (`stakeType`), so you can just reuse it and add to this callback's dependency array below instead of `getValues`.

Reply:
Good call - pushed a fix